qutebrowser/completion/completionwidget.py

```diff
@@ -80,6 +80,10 @@ class CompletionView(QTreeView):
         QTreeView:item::hover {
             border: 0px;
         }
+
+        QTreeView {
+            border: 0px;
+        }
```

Comment:
There is already a `QTreeView { ... }` section at the top - can you add it there instead?